modules/nw-multus-create-master-interface-bridge-cni.adoc

```diff
@@ -0,0 +1,212 @@
+
+.Procedure
+
+. Create a dedicated container namespace where you want to deploy your pod by running the following command:
++
+[source,terminal]
+----
+$ oc new-project test-namespace
+----
+
+. Create a bridge network attachment definition file named `bridge-nad.yaml` and paste the following content:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: bridge-network
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "name": "bridge-network",
+    "type": "bridge",
+    "bridge": "br-001",
+    "isGateway": true,
+    "ipMasq": true,
+    "hairpinMode": true,
+    "ipam": {
+      "type": "host-local",
+      "subnet": "10.0.0.0/24",
+      "routes": [{"dst": "0.0.0.0/0"}]
+    }
+  }'
+----
+
+. Run the following command to apply the NetworkAttachmentDefinition to your OpenShift cluster:
++
+[source,terminal]
+----
+$ oc apply -f bridge-nad.yaml
+----
+
+. Verify that the NetworkAttachmentDefinition has been created successfully by running the following command:
++
+[source,terminal]
+----
+$ oc get network-attachment-definitions
+----
++
+.Expected output
+
+[source,terminal]
+----
+NAME             AGE
+bridge-network   15s
+----
+
+. Create a YAML file for the IPVLAN additional network configuration by saving the following YAML in a file named `ipvlan100-additional-network-configuration.yaml`:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ipvlan-100
+  namespace: test-namespace
+spec:
+  config: '{
+    "cniVersion": "0.3.1",
+    "name": "ipvlan-net",
+    "type": "ipvlan",
+    "master": "ext0", <1>
+    "mode": "l3",
+    "linkInContainer": true, <2>
+    "ipam": {"type": "whereabouts", "ipRanges": [{"range": "10.0.0.0/24"}]}
+  }'
+----
++
+<1> Specifies the Ethernet interface to associate with the network attachment. This is subsequently configured in the pod networks annotation.
+<2> Specifies that the master interface is in the container network namespace.
+
+. Apply the YAML by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ipvlan100-additional-network-configuration.yaml
+----
+
+. Verify that the NetworkAttachmentDefinition has been created successfully by running the following command:
++
+[source,terminal]
+----
+$ oc get network-attachment-definitions
+----
++
+.Expected output
+
+[source,terminal]
+----
+NAME             AGE
+bridge-network   9m36s
+ipvlan-100       9s
+----
+
+. Create a pod definition by using the following YAML. Save the YAML in a file named `pod-a.yaml`.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-a
+  namespace: test-namespace
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {
+        "name": "bridge-network",
+        "interface": "ext0" <1>
+      },
+      {
+        "name": "ipvlan-100",
+        "interface": "ext0.100"
```

Comment:
use another name. This gives the impression that the interface has vlan 100

Reply:
so something like ipvlan-network would do @mlguerrero12

Reply:
it could be
name: ipvlan-net
interface: ext1